### DISCUSSION TITLE
[Feature] to organize the tablist per groups

### DISCUSSION THREAD

**Discussion by marcangeloh** [Zrips/CMI#3479] [Feature Requests]
**Description of issue or feature request:**
Basically on my server we have different groups (such as owner, admin, mod, etc.) But when the tablist loads, (with the correct prefix), the names of the players aren't grouped up based on their rank. I have tried doing it with the cmi.tablist.[groupnumber] permission however it didn't work.

Love the plugin, keep up the great work!

**Comment by mrfloris** [comment]
@Zrips  This is almost a daily question on Discord. And a lot of people turn tablist off and use TAB or another plugin to resolve this request. I've seen this question in a variety of ways though. 

Bungee support for tablist wouldn't hurt either to be honest if it isn't already showing players from all servers.

**Comment by CloudeLecaw** [comment]
The tablist groupnumber permission you set is just to be able to have for each group a different design of the tab (if you have actived it) if a group has more then one numer then the highest numberdesign based on their permissions is shown. So players can have a different Tabdesign then Staff
